src/extension.ts

```diff
@@ -14,6 +14,8 @@ import { FileWatcher } from './helpers/fileWatcher';
 export async function activate(context: vscode.ExtensionContext) {
 
 	global.workspaceChain = Workspace.GetWorkspaceChainConfig();
+	global.workspaceChain.coinType = global.workspaceChain.coinType || "118";
```

Comment:
Can you set coinType to workspaceChain like signType?

Reply:
I have already set coinType like signType in config.json. But because the default value cannot work, so I hardcode here. (The same as signType and chainEnvironment, and this is #6 )

Reply:
https://github.com/Finschia/cosmy-wasmy/blob/6dcc26544fcdf09ff64660884d575d732dff5ce6/src/helpers/sign/wrapwallet.ts#L18
Cosmy Wasmy seems to handle dafault values of signType in Wrapwallet.

Reply:
But I think it is better to set default value in global.workspaceChain rather than in wrap wallet. Not each field in configuration is used to initial wrap wallet, like ```chainEnvironment```.

Reply:
Why is it better to set it in global?

It makes the cause of bugs because it requires default value handling in every global.

Reply:
Ok, I will make another PR to summarize the default value.